src/Directory.Packages.props

```diff
@@ -0,0 +1,38 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+    <NoWarn>$(NoWarn);NU1507</NoWarn>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Azure.Bicep.Core" Version="0.25.53" />
+    <PackageVersion Include="Azure.Deployments.Core" Version="1.0.1158" />
+    <PackageVersion Include="Azure.Deployments.Expression" Version="1.0.1158" />
+    <PackageVersion Include="Azure.Deployments.Templates" Version="1.0.1158" />
+    <PackageVersion Include="Azure.Identity" Version="1.11.4"/>
+    <PackageVersion Include="coverlet.collector" Version="3.0.2" />
+    <PackageVersion Include="FluentAssertions" Version="6.2.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging" Version="6.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.1" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
+    <PackageVersion Include="Microsoft.PowerPlatform.ResourceStack" Version="6.0.0.1485" />
+    <PackageVersion Include="Microsoft.PowerShell.SDK" Version="7.4.1" />
+    <PackageVersion Include="Microsoft.PSRule.Rules.Azure" Version="1.33.2" />
+    <PackageVersion Include="Microsoft.PSRule.SDK" Version="2.9.0" />
```

Comment:
Looks like these weren't updated in Analyzer.Core.nuspec previously. Can you make that update?

Reply:
Sarif SDK and System.IO.Abstractions need an update too